bait-files/src/main/webapp/SubmissionPage.html

```diff
@@ -0,0 +1,50 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Profile Submission Page</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="submissionPage.css">
+  </head>
+  <body>
+      <h3>What do you know about this person?</h3>
+      <p>Fill out as many fields as you can with the information that you have.</p>
+      <form action="/manual-review" method="post">
```

Comment:
Should the contents of this form be going straight to manual review? Maybe rename the action and accompanying servlet to something more general?

Reply:
Agreed =)  the action of a form should generally be the name of the API endpoint that it will be calling on the backend.
So you could go with something like "/start-verification" or "/submit-information" or something.

Alternatively, you could go with more traditional REST API naming (https://cloud.google.com/apis/design/resources#what_is_a_rest_api).
As a brief background, when using REST conventions, you think of every endpoint as the management of some "thing" that can be named as a noun, and so you name every endpoint as the plural of a noun.

In this case, you'd want to come up with a name for what "thing" you're creating when you submit information about a person.  It seems to me like what is being created in this case is a verification / investigation process, so you could call it something like "/verifications" or "/investigations".  Under this pattern you would send a POST request when you want to create a new verification/investigation and send a GET request to the same endpoint when you want to retrieve the result.

Anyway, you don't need to completely figure the name out in this PR since either way there is no server to answer the request, but it could be good to start thinking along one of the lines I've mentioned.